client/scripts/views/components/proposals/aave_proposal_card_detail.ts

```diff
@@ -10,14 +10,16 @@ export const roundVote = (percentage) => {
 
 const AaveProposalCardDetail = {
   view: (vnode: m.Vnode<{ proposal: AaveProposal, statusClass: string, statusText: any }>) => {
-    const { proposal, statusClass, statusText } = vnode.attrs;
+    const { proposal, statusClass } = vnode.attrs;
+    const statusText = Array.isArray(vnode.attrs.statusText)
```

Comment:
What happened here, with the sometimes array?

Reply:
Those countdowns are basically just timers -- we may want to remove the countdowns from Aave proposals only, but we definitely will still want them for Substrate stuff. For now, this is fine, though.